dev/tests/static/phpunit.xml.dist

```diff
@@ -30,6 +30,7 @@
     </testsuites>
     <php>
         <ini name="date.timezone" value="America/Los_Angeles"/>
+        <env name="phpunit.static.testcodestyle.use_whitelist" value="0"/>
```

Comment:
I suggest changing it to `testcodestyle.is_full_scan` and default `1` value.

I'm not sure whether it's better to use env or const, like we do in integration tests `<const name="TESTS_CLEANUP" value="enabled"/>`.